### ISSUE TITLE
Issue with docs.zowe.org/stable/user-guide/start-zowe-zos/

### ISSUE THREAD

**Issue by dkelosky** [zowe/docs-site#3530]
## Description

This page states:

>With Zowe version 1, you can issue C command to stop Zowe main server. This is not supported in version 2 anymore. A P command is required to make sure Zowe components can be shuted down properly.

* "shuted" is a typo
* `C` is a `CANCEL` command which would "stop" Zowe server, but it's confusing to state that you issue a `CANCEL` command to "stop" as that mixes up two distinct MVS system commands.
* Is this alert really necessary? Was `STOP` not accepted in the prior release?  If it was, I think this alert could be deleted entirely as we would just direct people to use the `STOP` command.

## Pages to Update

Issue with docs.zowe.org/stable/user-guide/start-zowe-zos/

**Comment by Martin-Zeithaml** [comment]
>With Zowe version 1, you can issue C command to stop Zowe main server. This command is no longer supported in version 2. The P command is now required to ensure that the Zowe components shut down properly.

I agree. This is confusing. It almost seems, what we have written MVS command `C`, which is not supported in V2 :smile:

Doc and state for V1:
* `zowe-stop.sh` is using `CANCEL`
  * I don't know, what is the reason to use cancel for stop
* Suggested way is `CANCEL` for `ZWESVSTC` (Zowe Main Task)
* And `STOP` for `ZWESLSTC` (launcher)

Based on the doc and code, this is simple and true:

In version 1, use `C` command for `ZWESVSTC` (Zowe Main Server).
Regardless the version, use `P` command for the `ZWESLSTC` (Zowe Launcher).
